recipe/build.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+# skip the creation of man pages by faking existance of help2man
+if [ `uname` == Darwin ]; then
+    export HELP2MAN=/usr/bin/true
+fi
+if [ `uname` == Linux ]; then
+    export HELP2MAN=/bin/true
```

Comment:
Yes, I decided it was more difficult to get this to work and opted to just skip the man pages.  Thanks for opening the issue so that this change can be tracked.